Description: Don't ship embedded MathJax JS library in Debian's atril package
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/backend/epub/Makefile.am
+++ b/backend/epub/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = minizip MathJax
+SUBDIRS = minizip
 
 AM_CPPFLAGS = 		       			\
 	-I$(top_srcdir)				\
--- a/configure.ac
+++ b/configure.ac
@@ -810,58 +810,6 @@
 backend/dvi/mdvi-lib/Makefile
 backend/epub/Makefile
 backend/epub/minizip/Makefile
-backend/epub/MathJax/Makefile
-backend/epub/MathJax/config/Makefile
-backend/epub/MathJax/config/local/Makefile
-backend/epub/MathJax/extensions/Makefile
-backend/epub/MathJax/extensions/MathML/Makefile
-backend/epub/MathJax/extensions/TeX/Makefile
-backend/epub/MathJax/images/Makefile
-backend/epub/MathJax/jax/Makefile
-backend/epub/MathJax/jax/element/Makefile
-backend/epub/MathJax/jax/element/mml/Makefile
-backend/epub/MathJax/jax/element/mml/optable/Makefile
-backend/epub/MathJax/jax/input/Makefile
-backend/epub/MathJax/jax/input/AsciiMath/Makefile
-backend/epub/MathJax/jax/input/TeX/Makefile
-backend/epub/MathJax/jax/input/MathML/Makefile
-backend/epub/MathJax/jax/input/MathML/entities/Makefile
-backend/epub/MathJax/jax/output/Makefile
-backend/epub/MathJax/jax/output/SVG/Makefile
-backend/epub/MathJax/jax/output/SVG/autoload/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/AMS/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/AMS/Regular/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Script/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Script/Regular/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Caligraphic/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Caligraphic/Bold/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Caligraphic/Regular/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Fraktur/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Fraktur/Regular/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Fraktur/Bold/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Math/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Math/Italic/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Math/BoldItalic/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Size1/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Size1/Regular/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Size2/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Size2/Regular/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Size3/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Size3/Regular/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Size4/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Size4/Regular/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Main/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Main/Regular/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Main/Bold/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Main/Italic/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/SansSerif/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/SansSerif/Bold/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/SansSerif/Italic/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/SansSerif/Regular/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Typewriter/Makefile
-backend/epub/MathJax/jax/output/SVG/fonts/TeX/Typewriter/Regular/Makefile
 backend/pdf/Makefile
 backend/pixbuf/Makefile
 backend/ps/Makefile
--- a/backend/epub/epub-document.c
+++ b/backend/epub/epub-document.c
@@ -1642,8 +1642,7 @@
 {
 	gchar *containerfilename= g_filename_from_uri(containeruri,NULL,NULL);
 	const gchar *backenddir = ev_backends_manager_get_backends_dir();
-	GString *mathjaxdir = g_string_new(backenddir);
-	g_string_append_printf(mathjaxdir,"/epub/MathJax");
+	GString *mathjaxdir = g_string_new("/usr/share/javascript/mathjax");
 
 	gchar *mathjaxref = g_filename_to_uri(mathjaxdir->str,NULL,NULL);
 	gchar *nodedata = g_strdup_printf("%s/MathJax.js?config=TeX-AMS-MML_SVG",mathjaxref);
